packages/floating-label/index.tsx

```diff
@@ -104,8 +109,8 @@ export default class FloatingLabel extends React.Component<
 
   handleWidthChange = () => {
     const {handleWidthChange} = this.props;
-    if (handleWidthChange && this.labelElement_.current) {
-      handleWidthChange(this.labelElement_.current.offsetWidth);
+    if (handleWidthChange && this.labelElement.current) {
+      handleWidthChange(this.labelElement.current.offsetWidth);
```

Comment:
Can we directly refer `this.props.handleWidthChange` instead? I read this like `handleWidthChange` method calling by itself.